### ISSUE TITLE
Memory leaks (serious)

### ISSUE THREAD

**Issue by gnuheidix** [gnuheidix/dicomux#10]
We got big problems with our memory usage. The garbage collector doesn't really free the memory. Especially after opening a directory. There were some open file issue which are fixed imho.

This has to be checked!

**Comment by gnuheidix** [comment]
I tried to reduce the available memory for the JVM in order to force the GC to run more often. I used the VM argument -Xmx100m which limits the heap size to 100 megabytes. The memory usage is still over 100MB because the heap isn't the only memory in the JVM. But it stays below 300MB. After setting it to 30MB heap size, it stays below 100MB memory usage. Unfortunately, Dicomux won't be able to open the DicomAnonym directory. The stack trace says that the plug-in dies due to memory problems. Only the stack trace holds this information. The user has to be informed that something bad happened in a friendly way.
